lib/libraries/nunjucks.js

```diff
@@ -8,6 +8,7 @@ module.exports = (() => {
         'app/_layouts',
         'node_modules/govuk-frontend'
       ], {
+        noCache: true,
         watch: process.env.NODE_ENV === 'development'
```

Comment:
**Side note:** Running the npm scripts with the environment variable `NODE_ENV` set to 'development` caused the application to hang during the 11ty build.